cvat-core/src/project-implementation.ts

```diff
@@ -96,9 +98,14 @@ export default function implementProject(projectClass) {
         useDefaultSettings: boolean,
         targetStorage: Storage,
         customName?: string,
+        options?: { requestStatusCallback?: (request: Request) => void },
     ) {
-        const result = exportDataset(this, format, saveImages, useDefaultSettings, targetStorage, customName);
-        return result;
+        const { requestStatusCallback } = options || {};
+        const rqID = await exportDataset(this, format, saveImages, useDefaultSettings, targetStorage, customName);
+        if (rqID) {
+            return requestsManager.listen(rqID, { callback: requestStatusCallback });
+        }
+        return new Request({ status: RQStatus.FINISHED, message: '' });
```

Comment:
I believe the following approach is more correct:

On `cvat-core` level we do not want to run implicit listening inside `exportDataset`, `importDataset`, `exportBackup`, `importBackup`. These methods should return `Request` instance. If `cvat-ui` wants to listen for them, it calls `listen` explicitly. This way we will get rid of new options, like `requestStatusCallback` and it will make code easier. 

Need to think additionally about task creating. For simplicity, my basic idea is to split between `create` and `save`  (as the first one is worker process, the second one is quicker):
- `create` returns `Request` instance.
- `save` returns `Task` instance as before.

It is not unified with for example Project.save or with Issue.save. But maybe we do not need this unification as these processes are not performed in Workers.

Reply:
For now, never mind, let's discuss after full code review, when I have mode understanding

Reply:
Implemented as we discussed